tools/gradle/plugins/builder/src/main/groovy/network/ockam/gradle/builders/BuildersPlugin.groovy

```diff
@@ -68,8 +68,8 @@ class BuildersPlugin implements Plugin<Project> {
   }
 
   // starting from the current directory, look for a parent directory that has a Vagrantfile
-  static String findVagrantfileDir() {
-    def path = new File('.')
+  static String findVagrantfileDir(File currentDir) {
+    def path = new File(currentDir.getAbsolutePath())
```

Comment:
this was fixed in develop https://github.com/ockam-network/ockam/commit/a16cda2ee7709922d5d557a8722c8a7ce1143335